portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,21 +41,24 @@
 import javax.cache.CacheManager;
 import javax.cache.CacheStatistics;
 import java.util.Collections;
+import java.util.List;
 
 /** Servlet that returns comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  /**
-   * Retrieves comments from datastore, adds to array, and serves as JSON 
-   */
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //Retrieves query parameters
-    int numComments = Integer.parseInt(getParameter(request, "comments", "5"));
-    String language = getParameter(request, "language", "EN");
+  //Global variables
```

Comment:
Nit, technically these are per-object, not global (neither as visibility, nor as instantiation)